src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -318,14 +318,25 @@ Shims the SSL_CTX_set_verify method.
 PALEXPORT void CryptoNative_SslCtxSetVerify(SSL_CTX* ctx, SslCtxSetVerifyCallback callback);
 
 /*
+Shims the SSL_set_verify method.
+*/
+PALEXPORT void CryptoNative_SslSetVerifyPeer(SSL* ssl);
+
+
+PALEXPORT int32_t  CryptoNative_SslSetData(SSL* ssl, void *ptr);
+PALEXPORT void* CryptoNative_SslGetData(SSL* ssl);
```

Comment:
They may not be stellar comments, but every other function in this header has some sort of comment.